docs/bosh/README.md

```diff
@@ -398,13 +451,41 @@ Now you have the infrastructure ready to deploy a BOSH director.
 terminal multiplexer such as `tmux` or `screen`.
 
    ```
-   bosh-init deploy manifest.yml
+   bosh2 create-env manifest.yml
    ```
 
-1. Target your BOSH environment:
+1. Use `vim` or `nano` to create a certificate named `ca_cert.pem`:
+
+    ```
```

Comment:
In this case, the `ca_cert.yml` does not appear in the director manifest that we already have. This is the certificate of the CA that signed the certificate the director will present.

I think we have a few options here:

- stick with hardcoding the certificate as I've done here.

- generate the certificate/key/ca_certificate and fill in the director manifest using `bosh2 interopolate`. This has the same number of distinct steps as copying in the existing `certs.yml`.

- entirely replace the erb using an [ops files](https://bosh.io/docs/cli-ops-files.html) combined with variables. This is a superset of the former point but it would be consistent. I think this would be optimal and maximizes the value of upgrading to `bosh2`.

Which would be preferable?

Github :(